src/components/Navigation/OnlyMine/FilterPopoverIcon.tsx

```diff
@@ -0,0 +1,75 @@
+import { Popover } from '@material-ui/core';
+import { fade, makeStyles, Theme } from '@material-ui/core/styles';
+import { interactiveTextColor } from 'components/Theme/constants';
+import * as React from 'react';
+
+const useStyles = makeStyles((theme: Theme) => {
+  const horizontalButtonPadding = theme.spacing(1.5);
+  return {
+    buttonIcon: {
+      marginLeft: theme.spacing(1),
+      marginRight: -horizontalButtonPadding / 2,
+    },
+    resetIcon: {
+      cursor: 'pointer',
+      '&:hover': {
+        color: fade(interactiveTextColor, 0.4),
+      },
+    },
+    popoverContent: {
+      border: `1px solid ${theme.palette.divider}`,
+      borderRadius: 4,
+      marginTop: theme.spacing(0.25),
+      padding: `${theme.spacing(2)}px ${theme.spacing(1.5)}px`,
```

Comment:
nit: `padding: theme.spacing(2, 1.5),`
should have the same value

Reply:
got it fixed